lib/tasks/download-assets.js

```diff
@@ -25,14 +31,14 @@ async function downloadAsset ({ url, directory }) {
   const file = createWriteStream(localFile)
 
   // download asset
-  const assetRequest = await fetch(url)
+  const assetRequest = await httpClient.get(url, { responseType: 'blob' })
 
-  if (!assetRequest.ok) {
+  if (assetRequest.status < 200 || assetRequest.status >= 300) {
     throw new Error(`error response status: ${assetRequest.status}`)
   }
```

Comment:
I believe axios throws errors for >=300 responses, wondering if this now needs a try/catch around the httpClient.get call